lib/std/io/reader.zig

```diff
@@ -298,3 +304,9 @@ test "Reader.isBytes" {
     testing.expectEqual(true, try reader.isBytes("foo"));
     testing.expectEqual(false, try reader.isBytes("qux"));
 }
+
+test "Reader.skipBytes" {
+    const reader = std.io.fixedBufferStream("foobar").reader();
+    try reader.skipBytes(3);
+    testing.expect(try reader.isBytes("bar"));
+}
```

Comment:
Would it make sense to add negative paths as well? In particular, requesting skipping beyond the underlying buffer, and skipping when the underlying cursor is already set at EOF?

Reply:
Yeah, I can add another `testing.expectError` for `error.EndOfStream` after the `isBytes` check.